src/VfacTmdb/Tmdb.php

```diff
@@ -353,4 +353,49 @@ public function checkOptionSessionId(array $options, array &$return) : void
             $return['session_id'] = trim($options['session_id']);
         }
     }
+
+    /**
+     * Check date option
+     * @param  string $option
+     * @return bool
+     * @author Steve Richter <steve@nerdbra.in>
+     */
+    public function checkOptionDate(string $option) : bool
+    {
+        $check = preg_match('/^([0-9]{4})-([0-9]{2}-[0-9]{2})$/', $option);
```

Comment:
I'm not fan of this date checking. All cases are not covered. For instance, with this gestion, the date "2020-02-31" is valid, but it is not :)
The regex check a structure not a validity.
I suggest a validation function like:
```php
function validateDate($date, $format = 'Y-m-d H:i:s')
{
    $d = DateTime::createFromFormat($format, $date);
    return $d && $d->format($format) == $date;
}
```

Reply:
I agree, something like this would be way better. I will adjust